src/Organizations/OrganizationCreate/OrganizationCreate.js

```diff
@@ -26,6 +26,15 @@ const INITIAL_VALUES = {
   interfaces: [],
   contacts: [],
   isVendor: false,
+  edi: {
+    vendorEdiType: '31B/US-SAN',
```

Comment:
Are you sure we need such default values for every organization?

Reply:
https://issues.folio.org/browse/UIORGS-203?focusedCommentId=87330&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-87330